.travis.yml

```
@@ -1,7 +1,6 @@
 dist: xenial
```

```suggestion
dist: focal
```
Should we also go with the latest LTS?